amplivelist/amp-live-list.go

```diff
@@ -51,16 +51,16 @@ var blogs []BlogItem
 func InitBlogs() {
 	blogs = make([]BlogItem, 0)
 	blogs = append(blogs,
-		createBlogEntry("A green landscape", "/img/landscape5.jpg"),
-		createBlogEntry("A mountain landscape", "/img/landscape6.jpg"),
-		createBlogEntry("A lake landscape", "/img/landscape1.jpg"),
-		createBlogEntry("A trees landscape", "/img/landscape2.jpg"),
-		createBlogEntry("A village landscape", "/img/landscape3.jpg"),
-		createBlogEntry("A canyon", "/img/landscape4.jpg"),
-		createBlogEntry("A beach", "/img/beach.jpg"),
-		createBlogEntry("Houses on the street", "/img/houses.jpg"),
-		createBlogEntry("Blue sea", "/img/sea.jpg"),
-		createBlogEntry("A ship", "/img/ship.jpg"))
+		createBlogEntry("A green landscape", "/img/landscape_hills.jpg"),
+		createBlogEntry("A mountain landscape", "/img/landscape_mountains.jpg"),
+		createBlogEntry("A lake landscape", "/img/landscape_lake.jpg"),
+		createBlogEntry("A trees landscape", "/img/landscape_trees.jpg"),
+		createBlogEntry("A village landscape", "/img/landscape_village.jpg"),
+		createBlogEntry("A canyon", "/img/landscape_canyon.jpg"),
+		createBlogEntry("A beach", "/img/landscape_beach.jpg"),
+		createBlogEntry("Houses on the street", "/img/landscape_houses.jpg"),
+		createBlogEntry("Blue sea", "/img/landscape_sea.jpg"),
+		createBlogEntry("A ship", "/img/landscape_ship.jpg"))
```

Comment:
nit: can you make the labels and image names a bit more accurate?

* beach -> desert (I'm pretty sure ;-) )
* A mountain landscape -> Mountains
* A village landscape -> Scattered houses
* A ship -> A sailing ship
* A lake landscape -> Road leading to a lake